clang/test/dpct/cmake_migation/case_006/expected.txt

```diff
@@ -0,0 +1,8 @@
+add_executable(cudasift ${cuda_sources} ${sources} OPTIONS -arch=sm_35)
```

Comment:
```suggestion
add_executable(cudasift ${cuda_sources} ${sources})
```
Can we remove `OPTIONS -arch=sm_35`? I think it is not necessary for SYCL.

Reply:
Added rule to remove